optimum/exporters/onnx/convert.py

```diff
@@ -199,6 +190,9 @@ def validate_model_outputs(
     # Compute outputs from the ONNX model
     onnx_outputs = session.run(onnx_named_outputs, onnx_inputs)
 
+    # Create onnxruntime output names for comparison with the reference model inputs
```

Comment:
the comment somewhat contradicts the docstring I think. I would rather say "filter the output names" or something, and maybe add a `logger.info`